data/stories.yml

```diff
@@ -28,3 +28,11 @@ stories:
   - action: utter_did_that_help
   - intent: deny
   - action: utter_goodbye
+
+- story: path covid
```

Comment:
this story doesn't add anything to the bot since you already have two rules around greet -> utter_greet and covid -> action_covid

never the less, it doesn't hurt to keep it in here

Reply:
```suggestion
- story: covid question path
```